lib/livebook_web/live/session_live/secrets_component.ex

```diff
@@ -1,15 +1,14 @@
 defmodule LivebookWeb.SessionLive.SecretsComponent do
   use LivebookWeb, :live_component
 
-  alias Livebook.Hubs.EnterpriseClient
   alias Livebook.Secrets.Secret
 
   @impl true
   def update(assigns, socket) do
     socket =
       socket
       |> assign(assigns)
-      |> assign(connected_hubs: Livebook.Hubs.get_connected_hubs())
+      |> assign(hubs: Livebook.Hubs.get_connected_hubs([:secrets]))
```

Comment:
I don't think it needs to be connected here too. :)

```suggestion
      |> assign(hubs: Livebook.Hubs.get_hubs([:secrets]))
```